spec/timezone.html

```diff
@@ -555,20 +555,29 @@ <h1>ToTemporalTimeZone ( _temporalTimeZoneLike_ )</h1>
 
     <emu-clause id="sec-temporal-getoffsetnanosecondsfor" aoid="GetOffsetNanosecondsFor">
       <h1>GetOffsetNanosecondsFor ( _timeZone_, _instant_ )</h1>
+      <p>
+        The abstract operation GetOffsetNanosecondsFor observably calls the given _timeZone_ Object's `getOffsetNanosecondsFor` method and returns its result as an integer number of nanoseconds equivalent to 24 hours or less.
+      </p>
       <emu-alg>
         1. Let _getOffsetNanosecondsFor_ be ? GetMethod(_timeZone_, *"getOffsetNanosecondsFor"*).
         1. If _getOffsetNanosecondsFor_ is *undefined*, set _getOffsetNanosecondsFor_ to %Temporal.TimeZone.prototype.getOffsetNanosecondsFor%.
         1. Let _offsetNanoseconds_ be ? Call(_getOffsetNanosecondsFor_, _timeZone_, « _instant_ »).
         1. If Type(_offsetNanoseconds_) is not Number, throw a *TypeError* exception.
```

Comment:
I didn't do that because it would be a normative change. The spec text currently says to throw a TypeError in that case, and this would change it to a RangeError.

Reply:
(confirmed offline)